website/pages/icon.mdx

```diff
@@ -24,8 +24,8 @@ basic interface icons, to add your icons,
 import { CIcon } from "@chakra-ui/vue";
 ```
 
-Use an icon by passing the `name` props. This name must match an icon key in
-`theme.icons`. By default, the icon inherits the fontSize and color of it's
+Use an icon bypassing the `name` props. This name must match an icon key in
```

Comment:
```suggestion
Use an icon by passing the `name` pros. This name must match an icon key in
```

Reply:
I do think props shoulds stay props right?

Reply:
Yes props should remain props.